test/box/gh-5610-dirty-restart.test.lua

```diff
@@ -0,0 +1,20 @@
+env = require('test_run')
+test_run = env.new()
+test_run:cmd("create server gh_5610_dirty_restart with script='box/gh-5610-dirty-restart.lua'")
+test_run:cmd("start server gh_5610_dirty_restart")
+test_run:cmd("switch gh_5610_dirty_restart")
+
+fiber = require('fiber')
+s = box.schema.space.create('test', {is_sync = true})
+i = s:create_index('pk')
+fiber.new(function() box.space.test:insert{1} end)
```

Comment:
Here it's better to write like this
_ = fiber.new(function() box.space.test:insert{1} end)
I see in test result
fiber.new(function() box.space.test:insert{1} end)
 ---
 - status: suspended
   name: lua
   id: 124
I am not shure that id is always equal to 124, so it's may be flaky test

Reply:
I try to start it 100 times on my local machine and it's fails!
] +++ var/rejects/box/gh-5610-dirty-restart.reject	Wed Apr 14 14:29:24 2021
[001] @@ -31,7 +31,7 @@
[001]   | ---
[001]   | - status: suspended
[001]   |   name: lua
[001] - |   id: 124
[001] + |   id: 126
[001]   | ...

Reply:
Thanks! applied your fix.